extensions/rich_text_components/base.py

```diff
@@ -36,6 +36,9 @@ class BaseRichTextComponent(object):
     # The category the rich-text component falls under in the repository.
     # Overridden in subclasses.
     category = ''
+    # The component's name in the CKEditor widget system.
+    # Overriden in subclasses.
```

Comment:
Just to check: if we ever change this, will it affect content that's already in the datastore?